python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -151,8 +139,8 @@ def execute_action(self, data):
                 entities=entities,
                 project=project_entity,
                 sys_path=sys.path,
-                base_configuration=self.BASE_CONFIG_URI,
-                engine_name=self.ENGINE_NAME,
+                base_configuration=constants.BASE_CONFIG_URI,
+                engine_name=sgtk.platform.constants.SHOTGUN_ENGINE_NAME,
```

Comment:
this is not part of the public core API, so in this case i would have kept the `ENGINE_NAME` constant in the class.